src/main/java/fr/inria/coming/changeminer/analyzer/commitAnalyzer/FineGrainDifftAnalyzer.java

```diff
@@ -69,6 +73,27 @@ public FineGrainDifftAnalyzer() {
 			if (diff != null) {
 				diffOfFiles.put(fileFromRevision.getName(), diff);
 			}
+
+			DiffRowGenerator generator = DiffRowGenerator.create()
+					.showInlineDiffs(true)
+					.inlineDiffByWord(true)
+					.oldTag(f -> "~")
+					.newTag(f -> "**")
+					.build();
+			List<DiffRow> rows = null;
+			try {
+				rows = generator.generateDiffRows(
+						Arrays.asList("This is a test senctence.", "This is the second line.", "And here is the finish."),
+						Arrays.asList("This is a test for diffutils.", "This is the second line."));
+			} catch (DiffException e) {
+				e.printStackTrace();
+			}
+
+			System.out.println("|original|new|");
```

Comment:
This snippet seems to provide better results. WDYT?
```
			DiffRowGenerator generator = DiffRowGenerator.create()
					.showInlineDiffs(false)
					.inlineDiffByWord(false)
                    .ignoreWhiteSpaces(true)
					.build();

			List<DiffRow> rows = null;
			try {
				rows = generator.generateDiffRows(
                        Arrays.stream(fileFromRevision.getPreviousVersion().split("\n")).collect(Collectors.toList()),
                        Arrays.stream(fileFromRevision.getNextVersion().split("\n")).collect(Collectors.toList()));
			} catch (DiffException e) {
				e.printStackTrace();
			}

            System.out.println("Diff of the revision");
			for (DiffRow row : rows) {
			   switch (row.getTag()) {
                   case INSERT:
                       System.out.println("+ " + row.getNewLine());
                       break;
                   case DELETE:
                       System.out.println("- " + row.getOldLine());
                       break;
                   case CHANGE:
                       System.out.println("- " + row.getOldLine());
                       System.out.println("+ " + row.getNewLine());
                       break;
               }
			}
		}
```

Reply:
this snippet looks good!